docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.9"
```

Comment:
We're using docker-compose to run multiple dependencies as docker images locally with a single command. Right now it's a single dependency (postgres) but in the future if we have more, they will be added here.